crates/networking/p2p/sync_manager.rs

```diff
@@ -109,6 +121,12 @@ impl SyncManager {
                     };
                     *sync_head
                 };
+                // Edge case: If we are resuming a sync process after a node restart, wait until the next fcu to start
+                if sync_head.is_zero() {
+                    info!("Resuming sync after node restart, waiting for next FCU");
+                    sleep(Duration::new(5, 0)).await;
```

Comment:
I think it's better to use `from_secs` to keep it obvious:
```suggestion
                    sleep(Duration::from_secs(5)).await;
```

Reply:
Nice catch!